core/src/main/scala/org/apache/spark/ui/jobs/AllJobsPage.scala

```diff
@@ -192,6 +201,32 @@ private[ui] class AllJobsPage(parent: JobsTab, store: AppStatusStore) extends We
       </a>
     </span> ++
     <div id="application-timeline" class="collapsed">
+      {
+        if (MAX_TIMELINE_JOBS < jobs.size) {
+          <div>
+            <strong>
+              This page has more than the maximum number of jobs that can be shown in the
```

Comment:
Just thinking -> Shall we have this alert in red colour since this is one of the scenario where we are showing the limited information

Reply:
I think we should focus on resolving this issue itself for now.
The timeline view for StagePage shows similar message so if it's necessary, let's change the color for all the warning message.